packages/dd-trace/test/startup-log.spec.js

```diff
@@ -5,10 +5,13 @@ require('./setup/tap')
 const os = require('os')
 const tracerVersion = require('../../../package.json').version
 const Config = require('../src/config')
+const SamplingRule = require('../src/sampling_rule')
+const assert = require('node:assert')
```

Comment:
Nit: I like Node.js core modules to be at the top when possible (we should have a linting rule for that)

```suggestion
const assert = require('node:assert')
const tracerVersion = require('../../../package.json').version
const Config = require('../src/config')
const SamplingRule = require('../src/sampling_rule')
```

Reply:
While I agree that it's easier to read it that way, I think we should just let a linter handle it with autofix at a later point.